tests/rollapp_upgrade_test.go

```diff
@@ -415,3 +417,503 @@ func TestRollappUpgradeNonStateBreaking_Wasm(t *testing.T) {
 	testutil.AssertBalance(t, ctx, rollapp1, rollappUserAddr, rollapp1.Config().Denom, walletAmount.Sub(transferData.Amount))
 	testutil.AssertBalance(t, ctx, dymension, dymensionUserAddr, rollappIBCDenom, transferAmount)
 }
+
+func TestRollapp_EVM_Upgrade(t *testing.T) {
```

Comment:
I think we should add another rollapp to check ibc transfer from rollapp1 to rollapp2 after rollapp1 upgrade success  (its good to check before upgrade) to make sure everything work fine

Reply:
Yes, I know that, but first I create a simple upgrade test case first and then create an issue to add conditions for the test later.